app/src/main/java/org/simple/clinic/protocol/ProtocolRepository.kt

```diff
@@ -163,4 +163,28 @@ class ProtocolRepository @Inject constructor(
     )
   }
 
+  fun dosagesForDrug(drugName: String, protocolUuid: UUID?): Observable<List<String>> {
```

Comment:
Just like how we've suffixed other functions in this class with `orDefault()`, can we do the same thing here?

Reply:
Adding `orDefault` here can be misleading.
We are not going to return default dosages here. There is no concept of default dosages.

We are just checking that if `protocolUuid` is `null` that means the drugs shown are default.
Hence, we just want to get the dosages for those drugs from the default list.